### PR TITLE
Added article on allowing access to multi-user event management

### LINKED ISSUE #32: How do I allow multi-user access to manage event?
Article on allowing multiple users to manage event.

### PR DESCRIPTION
**Pull request by arihantbansal**
Added article on allowing multi user event management as part of GCI task. 

Fixes #32

### CODE REVIEW

File: event-setup/multi-user-event-management.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Now, on your event dashboard, scroll down to manage roles section.
+
+![Manage roles](/event-setup/images/manage-roles.PNG)
+
+A popup window appears where you can specify the user email and role to add.
+
+![Add people](/event-setup/images/popup.PNG)
+
+Clcik on add people to save your progress.
```
Comment (Masquerade0097) [inline event-setup/multi-user-event-management.md]: Click.

Comment (arihantbansal) [reply event-setup/multi-user-event-management.md]: Oh sorry. Fixed the typo.

**Comment by arihantbansal** [comment]
I tried setting up Github pages and hosting the site for preview link but it doesn't seem to work, even though the code is correct.

**Comment by arihantbansal** [comment]
Updated. Please merge.

**Comment by abishekvashok** [comment]
Can't see 'em

**Comment by arihantbansal** [comment]
Please check now https://github.com/arihantbansal/support.eventyay.com/blob/master/event-setup/multi-user-event-management.md

**Pull request merged**